src/components/Projects/WBS/WBSDetail/EditTask/EditTaskModal.jsx

```diff
@@ -61,13 +61,13 @@ const EditTaskModal = props => {
   const [status, setStatus] = useState('false');
 
   // hour best
-  const [hoursBest, setHoursBest] = useState(0);
```

Comment:
Highlight the small changes for reviewing the code. Sorry for inconvenience. 
- Fixed the warnings caused by unnecessary changes.